docs/configuration.md

```diff
@@ -13,3 +13,11 @@ Certain features of the React application import variables from an [AppConfig](h
 ### Custom Fonts & Styles
 Fonts and css variables can be customized by modifying [fonts-custom.scss](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/css/_fonts-custom.scss) and
 [variables-custom.scss](https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/css/_variables-custom.scss).
+
+### Examples
```

Comment:
Given that these example are related to the Application Config, how about nesting these examples under the `Application Config` section above?

Reply:
OK, my original intention was to have the React example(s?) [where I now think one example may actually be sufficient] in one section covering all of React, as the tricky part really is the rebuilding steps which is common across AppConfig and Font/Styling.

Anyway I nested it, and collapsed titles. I think that can also work!